.drone.yml

```diff
@@ -1,25 +1,101 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
```

Comment:
Just a quick question, why no auto release step in .drone.yml ?

Reply:
that's a very good question that I don't know the answer to 😄 
I'll add it.

Reply:
done ✅